web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -427,6 +427,57 @@ describe("buildUserRoadmap", () => {
       });
     });
 
+    describe("planned renovation", () => {
```

Comment:
[pebble] can you change the describe statements to say 'does not add' instead of 'does not show' because the tests are addressing whether the task is being added to the roadmap or not